### ISSUE TITLE
`verifyJson` skips null values

### ISSUE THREAD

**Issue by mscottford** [approvals/ApprovalTests.Java#222]
`verifyJson` relies on the pretty printing support in `gson` via the `setPrettyPrinting()` method on `GsonBuilder`. However, this ignores `null` values by default. To include `null` values, the `serializeNulls()` method on `GsonBuilder` also needs to be called.

Here's a demonstration:

Given this test:

```java
@Test
void demonstrateNullIssueInVerifyJson() {
	Approvals.verifyJson("{ \"nonNullField\": true, \"nullField\": null }");
}
```

Actual output:
```json
{
  "nonNullField": true
}
```

Expected output:
```json
{
  "nonNullField": true,
  "nullField": null
}
```

I'm happy to help implement this, and I'd like to start a discussion about the best way to do so. I'm guessing this needs to be "fixed" by adding a configuration option to `Approvals`. Otherwise, there would be risk of breaking people's tests that depend on the default behavior.

I'm curious about what others think the best approach is. One idea that I have is to provide a mechanism for delegating the configuration of the `GsonBuilder` to a method that `Approvals` users can interact with. That would also remove the duplication of the `GsonBuilder` configuration that's present in [`JsonUtils.prettyPrint()`][1] and [`JsonUtils.asJson()`][2].

Also, it looks like `JsonUtils` is a third-party library, but the source is copied into the `approvaltest-utils` package. My changes that I have in mind would require making changes to that code. Should it still be considered third-party code at that point?

[1]: https://github.com/approvals/ApprovalTests.Java/blob/master/approvaltests-util/src/main/java/com/spun/util/JsonUtils.java#L20
[2]: https://github.com/approvals/ApprovalTests.Java/blob/master/approvaltests-util/src/main/java/com/spun/util/JsonUtils.java#L32

**Comment by LarsEckart** [comment]
Hey Scott, thank you for bringing this up, we'll get back to it in the new year :)

**Comment by LarsEckart** [comment]
resolved with release of 13.2.0
